DominicGBauer/information-access
# fix: Base URL fix

# Pull Request

## Scope
Bug fix for base URL

## Work Done
Added "s" to "http" in dataservice.js

## Steps to Test
check that dataservice.js has "https" not "http" in base URL string. Once pushed to develop. Check dev webpage for search result functionality.

## Screenshot
![image](https://user-images.githubusercontent.com/80747408/136958596-b97511a9-d1f8-4787-a90e-f721537c70f0.png)
